### DISCUSSION TITLE
How to do model distillation in dpgen?

### DISCUSSION THREAD

**Discussion by cloud-shl** [deepmodeling/dpgen#1686] [Q&A]
The example on the bohrium website provides how to use dpgen2 for model distillation. In the "fp" package, it can select "deepmd" and specify the teacher model. However, after querying the dpgen's documentation, I can not found the similar function. How to do model distillation in dpgen? Can anyone give me some examples? Thanks.

**Comment by njzjz** [comment]
Please go to dpgen2's repository https://github.com/deepmodeling/dpgen2
